demo/ocs/ocs.env

```diff
@@ -1,91 +1,96 @@
 # Copyright 2020 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
 
+# Add proxy and other information here.
+# Proxy to be added as below for hostname 'company-proxy.com' and ports 80 and 443:
+# -Dhttp.proxyHost=company-proxy.com -Dhttp.proxyPort=80 -Dhttps.proxyHost=company-proxy.com -Dhttps.proxyPort=443
+_JAVA_OPTIONS=
+
 # Application version used for health check request.
-application.version=1.8
+APPLICATION_VERSION=1.8
```

Comment:
Application version should roll over to 1.9

Reply:
Done.